core/generated/main.go

```diff
@@ -18,15 +20,42 @@ func main() {
 		log.Fatalf("Error while reading config file %s", err)
 	}
 
-	dbHost, okHost := viper.Get("DB_HOST").(string)
-	dbUser, okUser := viper.Get("DB_USER").(string)
-	dbPass, okPass := viper.Get("DB_PASS").(string)
-	dbName, okName := viper.Get("DB_NAME").(string)
-	dbPort, okPort := viper.Get("DB_PORT").(string)
-	if !okHost || !okUser || !okPass || !okName || !okPort {
-		log.Fatalf("Error while reading database config")
+	dcsEstuary, ok := viper.Get("DCS_ESTUARY").(string)
+	if !ok {
+		log.Fatalf("Error while reading Estuary database config")
 	}
 
+	dcsMetrics, ok := viper.Get("DCS_METRICS").(string)
+	if !ok {
+		log.Fatalf("Error while reading Metrics database config")
+	}
+
+	setupMainDb(dcsEstuary)
+	setupMetricsDb(dcsMetrics)
+}
+
+func setupMetricsDb(dsn string) {
+
+	fmt.Println(dsn)
+	// specify the output directory (default: "./query")
+	// ### if you want to query without context constrain, set mode gen.WithoutContext ###
+	g := gen.NewGenerator(gen.Config{
+		OutPath: "generated/query",
+		Mode:    gen.WithoutContext | gen.WithDefaultQuery | gen.WithQueryInterface,
+	})
+	db, _ := gorm.Open(postgres.Open(dsn), &gorm.Config{})
+	g.UseDB(db)
+
+	// @jcace
+	// For first run - comment out the "ApplyBasic" lines, and only run GenerateModel
```

Comment:
@alvin-reyes just confirming you had to do this too, right? If we don't do this then the models can't be found

Reply:
Yes I had to do this.